<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionProfileDialog</class>
 <widget class="QDialog" name="ConnectionProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Профили</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string notr="true">Профили</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QListView" name="profileListView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string notr="true">Имя профиля:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="profileNameEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string notr="true">Соединение</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="4" column="1" colspan="2">
                <widget class="QLineEdit" name="schemaEdit"/>
               </item>
               <item row="1" column="1" colspan="3">
                <widget class="QLineEdit" name="userEdit"/>
               </item>
               <item row="0" column="3">
                <widget class="QLineEdit" name="portEdit"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string notr="true">Схема:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="3">
                <widget class="QLineEdit" name="passwordEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="serverEdit"/>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string notr="true">Порт:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string notr="true">База данных:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string notr="true">Сервер:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string notr="true">Пароль:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string notr="true">Логин:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="3">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string notr="true">Сбросить</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1" colspan="3">
                <widget class="QLineEdit" name="databaseEdit"/>
               </item>
               <item row="5" column="0" colspan="4">
                <widget class="QLabel" name="testResultLabel">
                 <property name="text">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>250</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string notr="true">Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string notr="true">Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="testButton">
         <property name="text">
          <string notr="true">Тест</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string notr="true">Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string notr="true">Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
